build.sbt

```diff
@@ -15,12 +15,12 @@ lazy val root = project
 
 val quasarVersion = IO.read(file("./quasar-version")).trim
 
-val http4sVersion = "0.20.10"
+val http4sVersion = "0.20.0-M5"
 val scalaXmlVersion = "1.1.0"
 
-val catsEffectVersion = "1.3.1"
+val catsEffectVersion = "1.1.0"
```

Comment:
Is downgrading `cats-effect` and `specs` also necessary? If not can we preserve them?

Reply:
I will upgrade `cats-effect` and `specs` separately tomorrow, I just wanted to revert the full PR.